packages/patternfly-react/src/components/VerticalNav/VerticalNav.stories.js

```diff
@@ -263,3 +264,23 @@ stories.add(
     </MockFixedLayout>
   ))
 );
+
+stories.add(
+  'With Masthead component',
+  withInfo({
+    propTablesExclude: [MockFixedLayout],
+    text: `Example using the **Masthead** component.`
+  })(() => (
+    <MockFixedLayout>
+      <div className="layout-pf layout-pf-fixed faux-layout">
+        <VerticalNav
+          sessionKey="storybookCustomMasthead"
+          items={mockNavItems}
+          masthead={<MastheadPf />}
+          showBadges
+        />
```

Comment:
How can I set navCollapsed prop in VerticalNav with the handle onNavToggleClick of MastHeadPf ?

Reply:
@aljesusg I'll take a stab at and example for this.

Reply:
Thanks @jeff-phillips-18 I read the documentation but I didn't find anything, the only solution is create a class mock with an state but I think that is not clean.

Reply:
If you don't want to create a stateful mock, I guess you could create two stories, with and without the `navCollapsed`.